lib/inferno/dsl/fhir_client.rb

```diff
@@ -60,26 +60,59 @@ def fhir_clients
         @fhir_clients ||= {}
       end
 
+      # Move parameters from body to the path for GET operations
+      #
+      # @param path [String]
+      # @param body [FHIR::Parameters] Must all be primitive if making GET request
+      # @private
+      def body_to_path(body)
+        query_hash = body.parameter.reduce({}) do |query, param|
```

Comment:
This doesn't handle repeated parameters. Probably easiest to stringify each param one by one and them join them together.

```ruby
query_strings =
  body.parameter.map do |param|
    # determine the key & value
    { key: value }
  end
query_string = "#{query_strings.map(&:to_query).join('&')}"
```

Reply:
This change caused some Cyclomatic complexity issues, so there is a new helper method called "safe_for_get?" that is used to simplify the inside of this method.